script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -276,22 +276,22 @@ The woman was leaving.
 You should go back to your room and do exactly as the doctor says."
 <Page76>
 "Ah―――"
-The woman walked away as if it was a matter of course.
+The woman walked away as if it were a matter of course.
 <Page77>
 "...See you tomorrow."
-Tomorrow we can talk again, just like today.
-I was glad. For the first time since I woke up from the accident, I felt human again.
+Tomorrow we can talk again, just like today. //Tense, self talking, single quite?
+I was glad. For the first time since I had woken up from the accident, I felt human again.
 <Page78>
 It became routine for me to go to the field in the afternoon.
 The woman would get upset if I called her Aoko. She said she didn't like her name.
 After much pondering, I decided to call her 'Sensei' because she seemed to be an important person.
 <Page79>
 Sensei listened to me seriously no matter what I said and solved my problems with just a few words.
 <Page80>
-After the accident, I had become uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
+After the accident, I had grown uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
 <Page81>
 Only when I was in that field did my anxiety disappear, because I knew that I could be my normal self there.
-Even though I didn't know who she was, she might really be a teacher at some school.
+Even though I didn't know who she was, she might really be a teacher at some school. //だから、どこの誰だかは知らないけれど、先生は本当にどこかの学校の先生なのかもしれない。 To retl
```

Comment:
> Because I didn't know who the heck she was. She might've really been a teacher somewhere.

Reply:
It's not clear what the `because` is linking to here.

Reply:
it's a continuation of
> because I knew that I could be my normal self there.

from the previous line. They're reasons for his lack of anxiety. Basically he's got a confidant who is a complete stranger imo

Reply:
Hm, if that's the case I might change prev line to 
> Only when I was in that field did my anxiety disappear. Because I knew that I could be my normal self there.

to make it more clear

Reply:
That's fine with me